client/my-sites/domains/domain-management/list/domain-row.jsx

```diff
@@ -170,14 +209,14 @@ class DomainRow extends PureComponent {
 
 	/* eslint-disable jsx-a11y/anchor-is-valid */
 	renderEmailLabel = () => {
-		const { domainDetails, translate } = this.props;
+		const { domain, translate } = this.props;
 
-		if ( [ domainTypes.MAPPED, domainTypes.REGISTERED ].indexOf( domainDetails.type ) === -1 ) {
+		if ( [ domainTypes.MAPPED, domainTypes.REGISTERED ].indexOf( domain.type ) === -1 ) {
```

Comment:
I know this was like this before, but this could become more expressive with
```
! [ domainTypes.MAPPED, domainTypes.REGISTERED ].includes( domain.type )
```